kusto/ingest/internal/resources/resources.go

```diff
@@ -57,10 +57,6 @@ func parse(uri string) (*URI, error) {
 		return nil, fmt.Errorf("URI scheme must be 'https', was '%s'", u.Scheme)
 	}
 
-	if !strings.HasSuffix(u.Hostname(), ".core.windows.net") {
-		return nil, fmt.Errorf("URI hostname does not end with '.core.windows.net'")
-	}
-
 	hostSplit := strings.Split(u.Hostname(), ".")
 	if len(hostSplit) != 5 {
 		return nil, fmt.Errorf("URI(%s) is invalid: had incorrect URL path before '.core.windows.net'", uri)
```

Comment:
This comment is incorrect

```suggestion
		return nil, fmt.Errorf("Storage URI (%s) is invalid'", uri)
```